src/server/style-import-hook.js

```diff
@@ -0,0 +1,48 @@
+import postcss from 'postcss';
+import Values from 'postcss-modules-values';
+import LocalByDefault from 'postcss-modules-local-by-default';
+import ExtractImports from 'postcss-modules-extract-imports';
+import Scope from 'postcss-modules-scope';
+import Parser from 'postcss-modules-parser';
+import genericNames from 'generic-names';
+import stylus from 'stylus';
+import { readFileSync } from 'fs';
+
+let result;
+
+// Require hook
+function hook(compile, extension) {
```

Comment:
Proper docs would be nice. Especially when there is so much black magic going on in this project 😄